front/components/members/InvitationModal.tsx

```diff
@@ -342,6 +342,7 @@ export function EditInvitationModal({
               variant="primaryWarning"
               label="Revoke invitation"
               icon={XMarkIcon}
+              disabled={owner.ssoEnforced}
```

Comment:
so we had it after all? :+1: 

Would add a tooltip here "As SSO is enforced on your workspace, member revocation can only be done through..."